src/test/java/io/vertx/test/core/TLSCert.java

```diff
@@ -164,7 +164,26 @@ TrustOptions getClientTrustOptions() {
     KeyCertOptions getClientKeyCertOptions() {
       return new PemKeyCertOptions().setKeyPath("tls/client-key.pem").setCertPath("tls/client-cert-ca.pem");
     }
-  };
+  },
+
+    FAKE() {
```

Comment:
the procedure for producing the fake-server-keystore.jks should be documented in _ssl.txt_